examples/tracing/otlp/clickhouse/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.7"
+
+services:
+  clickhouse:
+    image: clickhouse/clickhouse-server:latest
+    ports:
+      - "8123:8123"
+      - "9000:9000"
+    volumes:
+      - /tmp/clickhouse/data:/var/lib/clickhouse/
+      - /tmp/clickhouse/logs:/var/log/clickhouse-server/
```

Comment:
I get:

```
 ⠋ Container clickhouse-clickhouse-1  Creating                                                                                                                     0.0s 
Error response from daemon: invalid mount config for type "bind": bind source path does not exist: /host_mnt/private/tmp/clickhouse/data
```

Do we need the volumes mounted here at all?

Reply:
@markphelps we do not. That was leftover from things I was doing on my machine.